src/vue/components/inflections-table-wide.vue

```diff
@@ -121,7 +137,9 @@ export default {
 
   data: function () {
     return {
+      standardFormView: null,
       state: {
+        view: null,
```

Comment:
If I am right, we have a rendered view as a data property of the vue component, so when it is created and rendered it saves to the property  and won't be destroyed until the whole component would be destroyed.
So it would be placed in RAM (similiar to Vuex but without a direct access to it from any place) for the all time extension would be active (as we don't destroy the panel ever).
May be we should clear it at some time? And rerender when it would be asked for again?
What do you think?

Reply:
We're using `state.view` instead of `view` component's prop because inflection table data can come in two forms. For user-selected words it comes in a form a View instance that is placed into the `view` prop of the component. But for inflection browser tables it gets a standard form data object within the `standardFormData` component's prop.

Both types of tables renders the same way. So we assign either the View object from the `view`prop to the `state.view` or the View object generated from the `standardFormData`. We then use `state.view` to render the table in a Vue template in both cases. Also, the fact that the `state.view` is not empty is an indication that the view is ready to be used in the template. So the use of `state.view` is mostly a matter of convenience.

In case of the word-related inflection tables `state.view` does not occupy more space than the object in the `view` prop: both share the same reference. In case of inflection browser tables, however, `state.view` holds a newly generated object that takes some additional space. We can sack that object (it will free up to several megabytes of data) when the view is collapsed, but then we have to pay for that with up to several hundred milliseconds delay each time this table be opened again. Or we could have a near instant view display once it's rendered but have an overhead of several megabytes per each table. Which one is better? @balmas, what do you think?

Reply:
hmm. this is a tough call.  If I understand correctly, the additional space will be consumed only for those inflection browser tables that are actually viewed. Is that correct?  If so, then I think maybe the space consumed is worth the time saved because if someone is viewing inflection tables, I think they aren't likely to only look at them once.

Reply:
That is correct. The space will be taken only when the table is viewed: the expansion of the table view will trigger the rendering of its corresponding View object and the Vue-related DOM elements (because it uses `v-if`s).

Reply:
Ok, I don't think there's really a right answer here and it probably really varies depending upon how the application is being used. A sophisticated solution would probably adjust to usage, but I think that might be premature optimization. So let's just stick with keeping the object around, once created, choosing speed over memory consumption.